bin/tic_tac_toe

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/ruby
+
+require './lib/player'
```

Comment:
That's another `require_relative` just disguised now, isn't it? ;)
Please look up (and chat with other apprentices) about how to set up the `$LOAD_PATH` so that you don't have to hard code the exact directory layout.